test/ci-operator-integration/multi-stage/run.sh

```diff
@@ -30,8 +31,8 @@ if ! diff "${EXPECTED}" "${OUT}"; then
     exit 1
 fi
 
-# Run test with ci-operator-configresolver
-ci-operator-configresolver -config "${TEST_CONFIG_DIR}" -log-level debug -cycle 2m &
+# Run test with ci-operator-configresolver; registry isn't actually being used here, but needs to be set...
```

Comment:
Why not use it?

Reply:
This test part of the `ci-operator-integration` tests, so we're just testing that the ci-operator can communicate with the configresolver here. We actually test proper registry support in the `ci-operator-configresolver-integration` tests. In this case, the result shouldn't be any different with or without a populated registry, as all the steps in the confgi used here are literal steps.